features/steps/steps.py

```diff
@@ -12,8 +12,8 @@
 
 @given("a `{series}` lxd container with ubuntu-advantage-tools installed")
 def given_a_lxd_container(context, series):
-    if context.reuse_container:
-        context.container_name = context.reuse_container
+    if context.reuse_container and series in context.reuse_container:
```

Comment:
We can make this simpler
```suggestion
    if series in context.reuse_container:
    # OR 
     if context.reuse_container.get(series):
```

Reply:
Yes. addressed it, thanks